docs/index.bs

```diff
@@ -1575,12 +1575,14 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
       The <dfn method for="InstallEvent"><code>addRoutes(|rules|)</code></dfn> method steps are:
 
-        1. Let |routerRules| be a list of {{RouterRule}} dictionaries.
+        1. Let |serviceWorker| be the [=current global object=]'s associated [=ServiceWorkerGlobalScope/service worker=].
+        1. Let |routerRules| be a copy of |serviceWorker|'s [=list of router rules=].
```

Comment:
You may need to say "If |serviceWorker|'s [=list of router rules=] is set, " before "Let |routerRules|...".

Reply:
And, after the sentence, "Otherwise, initialize |routerRules| with an empty list."

Reply:
Upon the offline discussion, we decided to go with:
1. list of router rules is initialized with an empty list.
2. Making Fetch Handler Step 14 to ensure the list of router rules is not an empty list instead of confirming it is set.